plugins/modules/system/syspatch.py

```diff
@@ -6,26 +6,25 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
 module: syspatch
 
-short_description: Manage OpenBSD system patches
+short_description: Manage OpenBSD system patches.
```

Comment:
```suggestion
short_description: Manage OpenBSD system patches
```
we don't need a final dot in short descriptions